acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/FailureConditions.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.tests;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import tech.pegasys.ethsigner.jsonrpc.JsonRpcRequest;
+import tech.pegasys.ethsigner.jsonrpc.JsonRpcRequestId;
+
+import java.util.Collections;
+
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.http.HttpClientResponse;
+import io.vertx.core.json.Json;
+import org.junit.Test;
+
+public class FailureConditions extends AcceptanceTestBase {
+
+  private volatile HttpClientResponse response = null;
+
+  @Test
+  public void sendDisabledApiReturnsBadRequest() {
+    final JsonRpcRequest request = new JsonRpcRequest("2.0", "ibft_getPendingVotes", new String[0]);
+    request.setId(new JsonRpcRequestId(1));
+
+    ethSigner()
+        .sendRawJsonRpc(
+            Collections.emptyMap(),
+            Json.encodeToBuffer(request),
+            response -> this.response = response);
+
+    WaitUtils.waitFor(
+        1,
+        () -> assertThat(response.statusCode()).isEqualTo(HttpResponseStatus.BAD_REQUEST.code()));
+  }
+
+  @Test
+  public void unrecognisedJsonRpcMethodReturnsBadRequest() {
```

Comment:
nit: maybe unknown instead of unrecognised

Reply:
Done.